### ISSUE TITLE
Has this repo been repurposed?

### ISSUE THREAD

**Issue by mihirk** [Bahmni/bahmni-scripts#3]
The original intention of this was to run the same set of scripts on all go-agents.
The reason being to keep a version controlled records of who and how the agents were touched, and to execute the same script on all the agents. 

But from the latest commits it looks like it has been repurposed.

**Comment by gsluthra** [comment]
Yes. Seems like it. I didn't realize there was some restriction that all
go-agents will run the same scripts that are put here. If so, we can move
all the new scripts there. I will wait for someone from the team to add on.

Regards
Gurpreet

Community Lead, Bahmni
About me: http://life-lessons.in
